app/users.py

```diff
@@ -1,5 +1,40 @@
 """This module manages the users"""
+USERS = {
```

Comment:
Why do you have predefined users?

Reply:
Did that for running local tests, must have forgotten to delete that. Let me refactor.